extensions/query-history/package.nls.json

```diff
@@ -1,4 +1,5 @@
 {
 	"queryHistory.displayName": "Query History",
-	"queryHistory.description": "View and run previously executed queries"
+	"queryHistory.description": "View and run previously executed queries",
+	"queryHistory.clear": "Clear All History"
```

Comment:
Just checking - Udeesha mentioned issues with multiple uses of the same localizable string. We use this here and in the UI actions, should we just use different ID variants so there's no issue when producing the localization?

Reply:
@udeeshagautam - It's my understanding that the extensions are localized each separately so this shouldn't be an issue. 

(we're also using the same string here so it isn't a problem regardless, but even if they were different I don't think it'd matter)

I'll update to be different just in case, but it'd be good to verify if this would ever be an actual concern.